src/common/src/types/to_sql.rs

```diff
@@ -15,37 +15,35 @@
 use std::error::Error;
 
 use bytes::BytesMut;
-use postgres_types::{accepts, to_sql_checked, IsNull, ToSql, Type};
+use postgres_types::{to_sql_checked, IsNull, ToSql, Type};
 
-use crate::types::{JsonbRef, ScalarRefImpl};
+use crate::types::ScalarImpl;
 
-impl ToSql for ScalarRefImpl<'_> {
+impl ToSql for ScalarImpl {
```

Comment:
Most of our types that implement `FromSql` are `scalar` types instead of `scalar_ref`. So it's easier to implement `FromSql` for `ScalarImpl` then `ScalarRefImpl`. Besides, as the `postgres_cell_to_scalar_impl`'s return type is `ScalarImpl`, `ScalarImpl` would be more natural.